src/psyclone/tests/psyad/domain/lfric/test_lfric_adjoint.py

```diff
@@ -89,7 +88,8 @@ def test_generate_lfric_adjoint_no_routines_error(fortran_reader):
 """)
     with pytest.raises(InternalError) as err:
         generate_lfric_adjoint(psyir, ["var1", "var2"])
-    assert "The supplied PSyIR does not contain any routines" in str(err.value)
+    assert ("The supplied PSyIR does not contain any kernel metadata"
+            in str(err.value))
```

Comment:
Codecov says that now the previous error "not contain any routines" is not tested anywhere.

Reply:
Oops, well spotted. I vaguely remember being concerned about this when I modified the test but then I forgot to check and was fooled by CodeCov's summary saying everything was fine...ready for another look now.